src/qibocal/auto/task.py

```diff
@@ -86,6 +86,17 @@ def operation(self):
 
         return Operation[self.action.operation].value
 
+    def validate(self, results: Results):
+        if self.action.validator is not None:
```

Comment:
What do you return if `validator` is None?

Reply:
In practice `None`, because it's the default output of any Python function when it does not return.

A better layout might be:
```suggestion
        if self.action.validator is None:
            return None
```
i.e. an early return. And deindent the following.

If you'd annotate the return value as `Option[...]` it would even be beneficial.

Reply:
I agree.

Reply:
Fixed